blueprints/addon/files/.github/workflows/ci.yml

```
@@ -0,0 +1,65 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request: {}
+
+jobs:
+
+  lint:
+    name: "Lint"
```

Let's have this lint and `test:ember` (basic smoke tests) 
```suggestion
  test:
    name: "Tests"
```